.github/workflows/cicd-main.yml

```diff
@@ -5223,7 +5222,7 @@ jobs:
       - L0_Unit_Tests_GPU_Hydra
       #- OPTIONAL_L0_Unit_Tests_GPU_Lightning
       - L0_Unit_Tests_GPU_Others
-      
+      - L0_Setup_Test_Data_And_Models
```

Comment:
Do we really need this line? Is it a side job, so we dont need list it here ?

Other than that , LGTM

Reply:
Oh, if it’s a side job, maybe we should make sure it runs before all other tests to prevent undesired failures? If this prepares models that other tests expects, we could have random failures depending on job-timing?

Reply:
The tests listed here are just so that we require them to pass . We have other tests that depend on this one , so i think we should be fine without explicitly specifying this one . Since the subsequent ones wont pass. 

So i think we can remove this line wdyt?

Reply:
Yes that makes sense, will remove it